app/rdf/query-search.ts

```diff
@@ -34,36 +31,6 @@ const makeInFilter = (varName: string, values: string[]) => {
     }`;
 };
 
-// It's a bit difficult ot access the types of the various sparql libraries
-const exampleSelectQuery = SELECT``;
-const exampleDescribeQuery = DESCRIBE``;
-type SelectQuery = typeof exampleSelectQuery;
-type DescribeQuery = typeof exampleDescribeQuery;
-type StreamOfQuad = Parameters<typeof fromStream>[1];
-
-const executeAndMeasure = async <T extends SelectQuery | DescribeQuery>(
-  client: T extends SelectQuery ? ParsingClient : StreamClient,
-  query: T
-): Promise<{
-  meta: RequestQueryMeta;
-  data: T extends SelectQuery ? unknown[] : StreamOfQuad;
-}> => {
-  const startTime = Date.now();
-  // @ts-ignore
-  const data = await query.execute(client.query, {
-    operation: "postUrlencoded",
-  });
-  const endTime = Date.now();
-  return {
-    meta: {
-      startTime,
-      endTime,
-      text: query.build(),
-    },
-    data,
-  };
-};
-
```

Comment:
What is defined somewhere else ?

Reply:
I think this might have been implemented before SPARQL clients were decorated? I was surprised that we send two SearchCubes queries every time, and I discovered that in fact we pushed the same query using this logic to the query we already saved using the timed SPARQL client 👀